### PR TITLE
Update inline code style

### PR DESCRIPTION
**Pull request by jyeakel**

### CODE REVIEW

File: znai-reactjs/src/theme/znai-dark/znai-dark.css [hunk 1]
```diff
@@ -30,7 +30,8 @@
 
     --znai-link-decoration: none;
 
-    --znai-inlined-code-background-color: #38444d;
+    --znai-inlined-code-background-color: #181e21;
+    --znai-inlined-code-text-color: #bbb;
```
Comment (MykolaGolubyev) [inline znai-reactjs/src/theme/znai-dark/znai-dark.css:34]:
```suggestion
    --znai-inlined-code-text-color: #aaa;
```

File: znai-reactjs/src/App.css [hunk 2]
```diff
@@ -174,7 +175,7 @@
     --znai-collapse-uncollapse-background-color: var(--znai-snippets-title-background-color);
 
     --znai-inversed-text-color: #bbbbbb;
-    --znai-inversed-background-color: #282f34;
+    --znai-inversed-background-color: #030303;
```
Comment (MykolaGolubyev) [inline znai-reactjs/src/App.css:178]: did you change it on purpose or was it left over from some experimenting?

File: znai-reactjs/src/App.css [hunk 3]
```diff
@@ -173,7 +175,7 @@
...
```
Comment (jyeakel) [inline znai-reactjs/src/App.css:178]:
```suggestion
    --znai-inversed-background-color: #282f34;
```

File: znai-reactjs/src/App.css (hunk 2 above)
Comment (jyeakel) [reply znai-reactjs/src/App.css]: Think I fixed.

File: znai-reactjs/src/App.css (hunk 3 above)
Comment (jyeakel) [inline znai-reactjs/src/App.css]:
```suggestion
    --znai-inversed-background-color: #282f34;
```

File: znai-reactjs/src/doc-elements/default-elements/Snippet.css [hunk 4]
```diff
@@ -24,9 +24,11 @@ pre {
 code {
     font-family: var(--znai-code-font-family);
     font-size: var(--znai-inlined-code-text-size);
-
+
```
Comment (MykolaGolubyev) [inline znai-reactjs/src/doc-elements/default-elements/Snippet.css:27]: please remove this extra line?

Comment (MykolaGolubyev) [reply znai-reactjs/src/doc-elements/default-elements/Snippet.css:27]: seems like extra spaces but lets ignore it

**Pull request merged**